modules/tls-profiles-kubernetes-configuring.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * security/tls-profiles.adoc
+
+[id="tls-profiles-kubernetes-configuring_{context}"]
+= Configuring the TLS security profile for the Kubernetes control plane
+
+To configure a TLS security profile for the Kubernetes control components (Kubernetes API server, the Kubernetes scheduler, and the Kubernetes controller manager), edit the `APIServer` custom resource (CR).  If a TLS profile is not configured, the default TLS profile is `Intermediate`. The Kubernetes control plane consists of the Kubernetes API server, the Kubernetes scheduler, and the Kubernetes controller manager.
+
+.Sample `APIServer` CR that configures the `Old` TLS security profile
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: APIServer
+ ...
+spec:
+  tlsSecurityProfile:
+    old: {}
+    type: Old
+ ...
+----
+
+The TLS security profile defines the the minimum TLS version and the TLS ciphers required by the Kubernetes control plane components to communicate with each other.
+
+You can see the ciphers and the minimum TLS version of the configured TLS security profile  in the `APIServer` custom resource (CR) under `Spec.Tls Security Profile` and `Status.Tls Profile`. 
+
+[NOTE]
+====
+The Kubernetes control plane does not support TLS `1.3`.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Edit the default `APIServer` CR to configure the TLS security profile:
++
+[source,terminal]
+----
+$ oc edit APIServer cluster
+----
+
+. Add the `spec.tlsSecurityProfile` field:
++
+.Sample `APIServer` CR for a `Custom` profile
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: APIServer
+metadata:
+  name: cluster
+spec:
+  tlsSecurityProfile:
+    type: Custom <1>
+    custom: <2>
+      ciphers: <3>
+      - ECDHE-ECDSA-CHACHA20-POLY1305
+      - ECDHE-RSA-CHACHA20-POLY1305
+      - ECDHE-RSA-AES128-GCM-SHA256
+      - ECDHE-ECDSA-AES128-GCM-SHA256
+      minTLSVersion: VersionTLS11
+----
+<1> Specify the TLS security profile type (`Old`, `Intermediate`, or `Custom`). The default is `Intermediate`.
+<2> Specify:
+* `old: {}`
+* `intermediate: {}`
+* `custom:`
+<3> For the `custom` type, specify a list of TLS ciphers and minimum accepted TLS version.
+
+. Save the file to apply the changes. 
+
+.Verification
+
+Verify that the TLS security profile is set in the `APIServer` CR:
+
+[source,terminal]
+----
+$ oc describe apiserver cluster
+----
+
+.Example output
+[source,terminal]
+----
+Name:         cluster
+Namespace:
+Labels:       <none>
+Annotations:  include.release.openshift.io/ibm-cloud-managed: true
+              include.release.openshift.io/self-managed-high-availability: true
+              include.release.openshift.io/single-node-developer: true
+              oauth-apiserver.openshift.io/secure-token-storage: true
+              release.openshift.io/create-only: true
```

Comment:
I think we may want to exclude these annotations, they are not relevant to this documentation anyway